### ISSUE TITLE
doesnt work

### ISSUE THREAD

**Issue by 3052** [zcahana/palgate-sdk#1]
no way to get the AuthToken currently:

https://github.com/zcahana/palgate-sdk/blob/ccabf2958d2faf664e556baaf6db20df4b20845a/config.go#L24-L28

**Comment by zcahana** [comment]
Correct, the PalGate app's authentication flow has changed quite some time ago, and the attempts to reverse engineer and reconstruct the new flow had came up empty. See RoeiOfri/homebridge-palgate-opener#17 for some background if you're interested.
